# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: C:\Documents and Settings\mdkeijzer\Mijn documenten\zendframework_svn\library\Zend\Captcha
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: Image.php
--- Image.php Base (BASE)
+++ Image.php Locally Modified (Based On LOCAL)
@@ -595,6 +595,6 @@
      */
     public function render(Zend_View_Interface $view = null, $element = null)
     {
-        return '<img width="'.$this->getWidth().'" height="'.$this->getHeight().'" alt="'.$this->getImgAlt().'" src="' . $this->getImgUrl() . $this->getId() . $this->getSuffix() . '"/><br/>';
+        return '<img width="'.$this->getWidth().'" height="'.$this->getHeight().'" alt="'.$this->getImgAlt().'" src="' . $this->getImgUrl() . $this->getId() . $this->getSuffix() . '"/>';
     }
 }
